src/ManagedServices/ManagedServices/Commands/GetAzureRmManagedServicesAssignment.cs

```diff
@@ -0,0 +1,79 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Commands
+{
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Models;
+    using System;
+    using System.Management.Automation;
+
+    [Cmdlet(
+        VerbsCommon.Get,
+        Microsoft.Azure.Commands.ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ManagedServicesAssignment",
+        DefaultParameterSetName = DefaultParameterSet,
+        SupportsShouldProcess = true), OutputType(typeof(PSRegistrationAssignment))]
+    public class GetAzureRmManagedServicesAssignment : ManagedServicesCmdletBase
+    {
+        protected const string DefaultParameterSet = "Default";
+
+        [Parameter(Position = 0, ParameterSetName = DefaultParameterSet, HelpMessage = "The subscription scope.")]
+        [ScopeCompleter]
+        public string Scope { get; set; }
+
+        [Parameter(Position = 1, ParameterSetName = DefaultParameterSet, ValueFromPipeline = true, HelpMessage = "The Registration Assignment Id.")]
+        [ValidateNotNullOrEmpty]
+        [Alias("Id")]
+        public string ResourceId { get; set; }
+
+        [Parameter(ParameterSetName = DefaultParameterSet, HelpMessage = "Whether to include the Registration Definiton in the response.")]
```

Comment:
This should be in both parameter sets, and so no parameter set name is necessary

Reply:
good catch